doctrine/common-toolbox.metadata.json

```diff
@@ -0,0 +1,21 @@
+{
+    "registrar": [
+        {
+            "language": "php",
+            "provider": "doctrine.models",
+            "signature": [
+                "Doctrine\\Common\\Persistence\\ManagerRegistry::getManagerForClass",
+                "Doctrine\\Common\\Persistence\\ObjectManager::find",
```

Comment:
what's about symfony plugin completion, does this results in duplicate entries in completion popover?